### PR TITLE
update polkadot api, add council test, update types

### PR DESCRIPTION
**Pull request by Leouarz**
Update types : To adapt to the new header version (from @kroos47, thx 😄)
Update polkadot api : To make council / democracy / technical committee work again
Added a test on council : To test governance feature from the api

### CODE REVIEW

File: tests/js_api/src/config.ts [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-    batch: 0,
-    count: 0,
-    amount: 0,
-    receiver: ''
-}
\ No newline at end of file
+export default {
+  mnemonic: 'bottom drive obey lake curtain smoke basket hold race lonely fit walk//Alice',
+  size: 0,
+  ApiURL: 'wss://https://testnet.avail.tools/ws',
```
Comment (kroos47) [inline tests/js_api/src/config.ts:15]: hey just asking, Do we really need to mention `https` here?

File: tests/js_api/src/api.ts [hunk 2]
```diff
@@ -77,9 +85,14 @@ export function createApi(): Promise<ApiPromise> {
...
                 appLookup: 'DataLookup'
             },
+            V2HeaderExtension: {
+                commitment: 'KateCommitmentV2',
+                appLookup: 'DataLookup'
+            },
             HeaderExtension: {
                 _enum: {
                     V1: 'V1HeaderExtension',
+                    V2: 'V2HeaderExtension',
```
Comment (prabal-banerjee) [inline tests/js_api/src/api.ts:95]: Are we dropping this for the testnet?

File: tests/js_api/src/api.ts [hunk 3]
```diff
@@ -7,6 +7,8 @@ export function createApi(): Promise<ApiPromise> {
     // Create the API and wait until ready
     return ApiPromise.create({
```
Comment (prabal-banerjee) [inline tests/js_api/src/api.ts:8]: @kroos47 Do you think we should use our wrapper here as well? This is ts and that is in js though.

**Review by prabal-banerjee — APPROVED** [review]
It appears to me that we have added scripts to `package.json` and hence there are better ways to run the script now.  This might need changes to the README. Kindly make sure that README is up to date.

File: tests/js_api/src/api.ts (hunk 2 above)
Comment (Leouarz) [reply tests/js_api/src/api.ts:95]: I am waiting for @kroos47 and @MiguelDD1 advice on that

**Comment by Leouarz** [comment]
Closing in favor of @0xSasaPrsic [PR](https://github.com/availproject/avail/pull/160)

**Pull request closed without merging**